webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/marshalling/XmlUnserialisationImpl.scala

```diff
@@ -793,6 +793,13 @@ class ApiAccountUnserialisationImpl extends ApiAccountUnserialisation {
     // mess up with authz
     import cats.implicits.*
 
+    def parse(s: String): Box[List[HttpAction]] = {
+      (s.split(",").map(_.trim).toList.filter(_.nonEmpty).traverse(HttpAction.parse)) match {
+        case Left(e)  => Failure(e)
+        case Right(x) => Full(x)
+      }
```

Comment:
for the sake of the already functional DSL :grin: (with `com.normation.errors.*` imported) :
```suggestion
      s.split(",").map(_.trim).toList.filter(_.nonEmpty).traverse(HttpAction.parse).toBox
```